guides/release/testing/index.md

```diff
@@ -205,16 +232,16 @@ The `--filter` option is more versatile. You can provide a phrase to match again
 
 ```bash
 # Button component example
-ember test --server --filter="should show icon and text"
+ember test --server --filter="should show icon and label"
 
 # Test everything related to your dashboard
-ember t -s -f="dashboard"
+ember t -s -f="Dashboard"
 
 # Run integration tests
-ember t -s -f="integration"
+ember t -s -f="Integration"
 ```
 
-With QUnit, you can also exclude tests by adding an exclamation point to the beginning of the filter, e.g. `ember test --filter="!acceptance"`.
+In QUnit, you can exclude tests by adding an exclamation point to the beginning of the filter, e.g. `ember test --filter="!Acceptance"`. In Mocha, `ember test --filter="Acceptance" --invert`.
```

Comment:
Mocha seems to do case-sensitive search for its `--invert` option. To help readers avoid possible confusion, I capitalized "integration" and "acceptance".